src/odemis/gui/cont/fastem_acq.py

```diff
@@ -251,12 +251,8 @@ def on_acquisition_done(self, future, num):
         try:
             da = future.result()
         except CancelledError:
-            self._reset_acquisition_gui()
             return
         except Exception:
-            # leave the gauge, to give a hint on what went wrong.
```

Comment:
Please add a comment that we do nothing, because it's all taken care by `full_acquisition_done()`.